Lib9c/Action/HackAndSlashSweep.cs

```diff
@@ -127,6 +143,32 @@ public override IAccountStateDelta Execute(IActionContext context)
                 );
             }
 
+            var equipmentList = avatarState.ValidateEquipmentsV2(equipments, context.BlockIndex);
+            var costumeIds = avatarState.ValidateCostume(costumes);
+            var items = equipments.Concat(costumes);
+            avatarState.EquipItems(items);
+            avatarState.ValidateItemRequirement(
+                costumeIds,
+                equipmentList,
+                sheets.GetSheet<ItemRequirementSheet>(),
+                sheets.GetSheet<EquipmentItemRecipeSheet>(),
+                sheets.GetSheet<EquipmentItemSubRecipeSheetV2>(),
+                sheets.GetSheet<EquipmentItemOptionSheet>(),
+                addressesHex);
+
+            var sweepRequiredCpSheet = sheets.GetSheet<SweepRequiredCPSheet>();
+            if (sweepRequiredCpSheet.TryGetValue(stageId, out var cpRow))
```

Comment:
If we execute sweep with a stageId does not contain in the sheet, it may succeed with 0 cp.